test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smdataparallel.py

```diff
@@ -95,6 +95,8 @@ def test_smdataparallel_mnist_script_mode_multigpu(ecr_image, sagemaker_regions,
     """
     Tests SM Distributed DataParallel single-node via script mode
     """
+    if ecr_image.split("/")[1].startswith("pr-pytorch-trcomp-training"):
```

Comment:
the ecr repo name changes in the rc build. use `endswith("pytorch-trcomp-training")`

Reply:
Also add this logic in `validate_or_skip_smdataparallel` function